### DISCUSSION TITLE
Family tree display for person records

### DISCUSSION THREAD

**Discussion by michael-wynne-wsu** [MukurtuCMS/Mukurtu-CMS#3] [New Feature Request]
Requested by multiple different users (infrequent).

Would like to have a view of person records arranged in a genealogical family tree - using the related person connections.

**Comment by MonicaGalassi** [comment]
@michael-wynne-wsu  thank you, that's a question we got as well. Also, I got asked by my team about a new template of the person record, as at the moment photos and text are not organised, etc. We all think this is a amazing feature and would like to be able to use it more

**Reply by michael-wynne-wsu** [reply]
> @michael-wynne-wsu thank you, that's a question we got as well. Also, I got asked by my team about a new template of the person record, as at the moment photos and text are not organised, etc. We all think this is a amazing feature and would like to be able to use it more

Monica, can you go add any requests/notes that you have about the existing person record tool to this discussion? #22 (I just created this right now) I'm not sure what you mean about organizing photos and text in this context, thanks!
